### PR TITLE
Adds screaming and a framework for gendered sounds for emotes.

### PR DESCRIPTION
**Pull request by chaoko99**
Why this is a draft: Female screams aren't working yet.

Known issues: Female screams don't work. All aliens will use human scream (I lack sounds to fix this anyways)
https://streamable.com/evs0l

### CODE REVIEW

File: code/modules/emotes/definitions/audible.dm [hunk 1]
```diff
@@ -3,10 +3,40 @@
...
+	var/male_sound //Give these strings for use through the sound.dm sound lists.
+	var/female_sound
+
+//Audible emotes with sound.
+client/
+	var/sound_cooldown = 0 //Can we do an emote with sound?
 
 /decl/emote/audible/do_extra(var/atom/user)
+	var/client/C = user.get_client()
+	if(isnull(C))
```
Comment (scrdest) [inline code/modules/emotes/definitions/audible.dm]: This call is overly-specific; if(!C) achieves the same thing and other edge-cases AFAIK.

File: code/modules/emotes/definitions/audible.dm [hunk 2]
```diff
@@ -3,10 +3,40 @@
...
+			switch(gen)
+				if("male")
+					gen = "male"
+				if("neuter")
+					gen = "male" //Functionality for neuterscreams is possible, but it'd require sounds.
+				if("female")
+					gen = "female"
+			playsound(user.loc, "[gen][key]" , 50, 0)
+		else
+			playsound(user.loc, emote_sound, 50, 0)
```
Comment (scrdest) [inline code/modules/emotes/definitions/audible.dm]: The whole block from L23 could be rewritten more cleanly with an assoc list. Something like: 
`var/list/soundmap = list("female"="female"); var/soundkey=soundmap[gen]; if(!soundkey) soundkey = male` (or go into the non-gendered block, your design call).

File: code/modules/emotes/emote_mob.dm [hunk 3]
```diff
@@ -149,3 +149,6 @@
 /mob/observer/ghost/emote(var/act, var/type, var/message)
 	if(message && act == "me")
 		communicate(/decl/communication_channel/dsay, client, message, /decl/dsay_communication/emote)
+
+
+
```
Comment (scrdest) [inline code/modules/emotes/emote_mob.dm]: Trim he.

Comment (chaoko99) [reply code/modules/emotes/emote_mob.dm]: Handled.

File: code/modules/emotes/definitions/audible.dm (hunk 2 above)
Comment (chaoko99) [reply code/modules/emotes/definitions/audible.dm]: Handled.

File: code/modules/emotes/definitions/audible.dm (hunk 1 above)
Comment (chaoko99) [reply code/modules/emotes/definitions/audible.dm]: Handled.

**Comment by chaoko99** [comment]
Ready for review.

**Pull request closed without merging**